### ISSUE TITLE
Build failure under Rust 0.11.0

### ISSUE THREAD

**Issue by kallisti5** [jimmycuadra/rust-etcd#13]
```
   Compiling etcd v0.5.4
/home/kallisti5/.cargo/registry/src/github.com-1ecc6299db9ec823/etcd-0.5.4/build.rs:27:37: 27:50 error: mismatched types [E0308]
/home/kallisti5/.cargo/registry/src/github.com-1ecc6299db9ec823/etcd-0.5.4/build.rs:27             serde_codegen::register(&mut registry);
                                                                                                                           ^~~~~~~~~~~~~
/home/kallisti5/.cargo/registry/src/github.com-1ecc6299db9ec823/etcd-0.5.4/build.rs:27:37: 27:50 help: run `rustc --explain E0308` to see a detailed explanation
/home/kallisti5/.cargo/registry/src/github.com-1ecc6299db9ec823/etcd-0.5.4/build.rs:27:37: 27:50 note: expected type `&mut syntex::Registry`
/home/kallisti5/.cargo/registry/src/github.com-1ecc6299db9ec823/etcd-0.5.4/build.rs:27:37: 27:50 note:    found type `&mut inner::syntex::Registry`
error: aborting due to previous error
error: Could not compile `etcd`.

To learn more, run the command again with --verbose.
```

**Comment by kallisti5** [comment]
oh... looks like a fix was already pulled.  Any chance of another release?

**Comment by kallisti5** [comment]
This Cargo.toml line is a temporary workaround:

```cargo
etcd = { git = "https://github.com/jimmycuadra/rust-etcd.git", branch = "master" }
```

**Comment by jimmycuadra** [comment]
Just released [v0.5.5](https://github.com/jimmycuadra/rust-etcd/releases/tag/v0.5.5).
